test/factories.rb

```diff
@@ -81,29 +89,22 @@
     zipcode { Faker::Address.zip_code }
 
     trait :with_adopter_profile do
-      after :create do |location|
-        create(:adopter_profile, location: location)
-      end
+      adopter_profile
     end
   end
 
   factory :organization do
     name { Faker::Company.name }
-    slug { Faker::Internet.domain_word }
-    profile { build(:organization_profile) }
+    sequence(:slug) { |n| Faker::Internet.domain_word + n.to_s }
```

Comment:
so we append a number here just to ensure it's unique? i.e., in the slim case Faker creates the same word twice.

Reply:
Exactly :) One of the tests I was debugging was sometimes failing because of this (but I can't remember now which one).